src/pytest_cov/plugin.py

```diff
@@ -226,6 +244,13 @@ def pytest_runtestloop(self, session):
                 compat_session.testsfailed += 1
 
     def pytest_terminal_summary(self, terminalreporter):
+        if self._disabled:
+            markup = {'red': True, 'bold': True}
+            msg = (
+                'WARNING: Coverage disabled by user!'
```

Comment:
I can only assume that you make this extra cruft here to make a longer message and use the pytest's warn system (in addition to writing to terminalreporter, of course)?

If so, you can use something like `terminalreporter.config.warn(code='COV-U1', message=msg)`.

Also, I would mention "--no-cov" somehow in the warning message, just to be clear what's going on.